Resources/Prototypes/Entities/Weapons/Ammunition/5.56mm/ammunition.yml

```diff
@@ -37,7 +37,7 @@
 
 - type: entity
   id: magazine_556mms
-  name: "5.56mm Magazine S"
+  name: "5.56mm magazine s"
```

Comment:
I think this S should probably stay capitalized? Since it obviously standards for something.